jobrunner/controller/main.py

```diff
@@ -348,7 +348,7 @@ def job_to_job_definition(job):
         image=full_image,
         args=action_args,
         env=env,
-        inputs=input_files,
```

Comment:
I wonder if we should use a different key, rather than changing the type of inputs? Thinking about tasks that are in-flight when this deploys, and potential breakage.

Or is the plan to cancel them all and get the controller to retry with the new job definition format post deploy?

Reply:
Yeah, a new key might be cleaner and more explicit here. Deployment is going to be a bit fiddly whichever way we cut it. But we don't have loads of activity at the moment so we may be able to get away without having to worry about in flight tasks.

Reply:
I'll make a new key, but I think if we wanted to handle in flight tasks we'd need to check in `prepare() ` for either `inputs` or `input_job_ids` (or whatever) and do the file-copying differently depending on which the task definition has. I'm not sure it's worth it - with not much activity going on, we could either wait until there are no jobs running before deploying, or run a prepare-for-reboot to cancel ongoing ones